### DISCUSSION TITLE
Unable to claim GitHub student benefits.

### DISCUSSION THREAD

**Discussion by rumiz123** [community/community#69264] [GitHub Education]
### Hi everyone,

I am unable to claim my github student benefits, I have been approved though

**Comment by thecogmel** [comment]
Me too

**Comment by mantasu** [comment]
Hey, I had the same issue, you just have to wait for a few more days as there will be some time before your benefits become active. When "Approved" tag becomes purple, benefits should be available:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

If you don't receive any benefits after 2 weeks, just open a [ticket](https://support.github.com/request/education?tags=docs-generic).

**Reply by rumiz123** [reply]
Ok, tag has been purple for around 2 days now

**Reply by rumiz123** [reply]
Still not working

**Reply by mantasu** [reply]
I would suggest waiting at least a week and then opening a ticket cuz this time is very busy with approvals

**Reply by mantasu** [reply]
Or you can open the ticket now and close it if you get access sooner

**Reply by rumiz123** [reply]
ok

**Reply by mantasu** [reply]
Worst case you could also try reapplying as suggested by the staff member [here](https://github.com/orgs/community/discussions/69133#discussioncomment-7185814)

**Reply by rumiz123** [reply]
ok

**Reply by rumiz123** [reply]
My support ticket got a response and they have reset it and told me to try again

**Reply by rumiz123** [reply]
Do you know how long one application lasts?

i mean before I have to submit another one?

**Reply by mantasu** [reply]
I think it's typically a week but on the site it says it can be several weeks. But if they asked to try again, I would just do so right away

**Reply by rumiz123** [reply]
Ok

**Reply by rumiz123** [reply]
I now have the pro badge but no benefits

**Reply by mantasu** [reply]
It might be some bug, definitely would suggest reapplying and re-raise the ticket if needed.

**Reply by rumiz123** [reply]
ok

**Reply by rumiz123** [reply]
I have access to the benefits but how do I link my github account to my jetbrains account?

**Reply by mantasu** [reply]
Congrats! For JetBrains you may have to apply manually with a student ID, see [here](https://www.jetbrains.com/lp/leaflets-gdc/students/).

**Reply by rumiz123** [reply]
Oh ok, Thanks

**Reply by mantasu** [reply]
No worries, enjoy your benefits :)

**Reply by rumiz123** [reply]
And about the microsoft azure benefits, how do I access those?

**Reply by mantasu** [reply]
For Azure, I think you should apply [here](https://azure.microsoft.com/en-us/pricing/offers/ms-azr-0170p/).

**Comment by muhammedfurkan** [comment]
same here :(

**Comment by P-Chaves** [comment]
Same!

**Comment by veje95** [comment]
same
